packages/redwood/src/index.ts

```diff
@@ -0,0 +1,143 @@
+import { join, dirname, relative, parse as parsePath, sep } from 'path';
+import {
+  BuildOptions,
+  Lambda,
+  Files,
+  PrepareCacheOptions,
+  createLambda,
+  download,
+  glob,
+  shouldServe,
+  debug,
+  getNodeVersion,
+  getSpawnOptions,
+  runNpmInstall,
+  spawnAsync,
+  execCommand,
+  FileBlob,
+  FileFsRef,
+} from '@vercel/build-utils';
+import { makeAwsLauncher } from './launcher';
+const {
+  getDependencies,
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+} = require('@netlify/zip-it-and-ship-it/src/dependencies.js');
+
+const LAUNCHER_FILENAME = '___vc_launcher';
+const BRIDGE_FILENAME = '___vc_bridge';
+const HELPERS_FILENAME = '___vc_helpers';
+const SOURCEMAP_SUPPORT_FILENAME = '__vc_sourcemap_support';
+
+export const version = 2;
+
+export async function build({
+  workPath,
+  files,
+  entrypoint,
+  meta = {},
+  config = {},
+}: BuildOptions) {
+  await download(files, workPath, meta);
+
+  const entrypointFsDirname = join(workPath, dirname(entrypoint));
+  const nodeVersion = await getNodeVersion(
+    entrypointFsDirname,
+    undefined,
+    config,
+    meta
+  );
+
+  const spawnOpts = getSpawnOptions(meta, nodeVersion);
+  await runNpmInstall(
+    entrypointFsDirname,
+    ['--prefer-offline'],
+    spawnOpts,
+    meta
+  );
+
+  debug('Running build script...');
+  await spawnAsync('yarn', ['rw', 'build'], {
+    ...spawnOpts,
+    cwd: workPath,
+    prettyCommand: 'yarn rw build',
+  });
+
+  const {
+    buildCommand = 'yarn rw db up --no-db-client --auto-approve && yarn rw build',
+  } = config;
```

Comment:
@AndyBitz Is `config.buildCommand` the correct way to get the project settings Build Command?

Reply:
Yes 👍